aspnetcore/mvc/razor-pages/index.md

```diff
@@ -223,9 +230,10 @@ services.AddMvc()
     .SetCompatibilityVersion(Microsoft.AspNetCore.Mvc.CompatibilityVersion.Version_2_1);
 ```
 
-`SetCompatibilityVersion` effectively sets the Razor Pages option `AllowMappingHeadRequestsToGetHandler` to `true`. The behavior is opt-in until the release of ASP.NET Core 3.0 Preview 1 or later. Each major version of ASP.NET Core adopts all of the patch release behaviors of the previous version.
+`SetCompatibilityVersion` effectively sets the Razor Pages option `AllowMappingHeadRequestsToGetHandler` to `true`.
+
+Rather than opting in to all 2.1 behaviors with `SetCompatibilityVersion`, you can explicitly opt-in to specific behaviors. The following code opts-in to the mapping HEAD requests to the GET handler.
```

Comment:
I see various remarks on phrasing "opting in to" vs. "opting into" ... I kind'a like the latter.

"opt-in" (verb) :point_right: ~"opt in"~ but might add the "to" :point_right: "opt into"

Same with the last "opts-in" (verb) ... "opts in" if not "opts into" as well.

> to the mapping HEAD requests to the GET handler.

Drop the "the" in that phrase?